interpreter/exec/simd.ml

```diff
@@ -48,6 +48,7 @@ sig
 
   val splat : lane -> t
   val extract_lane : int -> t -> lane
+  val extract_lane_u : int -> t -> lane
```

Comment:
If this is named `_u`, shouldn't the other be named `_s`?

Reply:
Good point, done!